### DISCUSSION TITLE
Next newsletter edition: 5 things you didn’t know you could do with GitHub Projects 😻

### DISCUSSION THREAD

**Discussion by tuves** [community/community#68206] [Announcements]
As a developer, you don't want to think about project management. 🤫💀 This month, we'll share our top 5 tips for using GitHub Projects to manage and track your work on GitHub without headaches. 🔥

[Don't miss it! Subscribe now](https://resources.github.com/newsletter/)

**Comment by Lailamoutafaddil** [comment]
Yes, of course, this is really what I want, thank you

**Reply by 2427-4** [reply]
อยากดูตัวอย่าง

**Comment by DRCLEMENTAKATENG** [comment]
How do I download and install Pi-hole from the GitHub site?

**Reply by 2427-4** [reply]
ผมจะสร้างคลิปได้อย่างใร

**Comment by 2427-4** [comment]
ไม่มี

**Comment by 2427-4** [comment]
Vedjo  look

**Comment by VerifiedShiraItay** [comment]
@2427-4 81202

**Comment by Liltex73** [comment]
> @2427-4 81202



> Vedjo look

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose). v
